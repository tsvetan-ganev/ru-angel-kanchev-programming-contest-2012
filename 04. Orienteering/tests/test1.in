25 2
1102Yordanov m 09:20:13 09:44:04
1702Nikolov m 08:30:15 08:44:00
2502Dimitrov m 09:19:28 09:42:27
0201Simeonov m 08:43:40 09:04:50
2301Galov m 08:25:14 08:42:05
0101Georgiev m 09:43:38 09:59:31
0301Milkov m 08:22:10 08:47:38
0401Doynov m 09:45:49 10:10:05
0701Bochukov m 10:58:56 11:21:13
2201Ivanov m 09:57:30 10:12:17
1801Zanev m 08:50:35 09:08:28
1502Penchev m 10:07:11 10:26:05
2001Kanev m 09:53:31 10:14:45
0502Ivanov m 10:56:07 11:18:34
1001Ovcharov m 10:46:35 11:00:04
2102Saratliev m 09:27:26 09:46:36
1601Ivanov m 09:04:19 09:29:23
0802Ivanov m 08:12:59 08:31:50
2402Zarev m 09:08:01 09:26:17
1201Tsenkov m 08:40:08 08:58:01
1402Mitev m 08:05:46 08:21:14
1302Krastev m 08:39:12 08:54:05
0902Manov m 10:41:05 10:57:34
0601Pavlov m 10:38:28 10:58:27
1902Vasilev m 10:03:23 10:26:59