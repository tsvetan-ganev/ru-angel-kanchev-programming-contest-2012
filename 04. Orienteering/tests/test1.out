10 17 22 

01 